docs/guides/setup/configuring-data-contexts/how-to-instantiate-a-data-context-without-a-yml-file.md

```diff
@@ -1,5 +1,206 @@
+                "batch_kwargs_generators": {},
+            }
+        },
+        stores={
+            "expectations_S3_store": {
+                "class_name": "ExpectationsStore",
+                "store_backend": {
+                    "class_name": "TupleS3StoreBackend",
+                    "bucket": "my_expectations_store_bucket",
+                    "prefix": "my_expectations_store_prefix",
+                },
+            },
+            "validations_S3_store": {
+                "class_name": "ValidationsStore",
+                "store_backend": {
+                    "class_name": "TupleS3StoreBackend",
+                    "bucket": "my_validations_store_bucket",
+                    "prefix": "my_validations_store_prefix",
+                },
+            },
+            "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+        },
+        expectations_store_name="expectations_S3_store",
+        validations_store_name="validations_S3_store",
+        evaluation_parameter_store_name="evaluation_parameter_store",
+        data_docs_sites={
+            "s3_site": {
+                "class_name": "SiteBuilder",
+                "store_backend": {
+                    "class_name": "TupleS3StoreBackend",
+                    "bucket":  "my_data_docs_bucket",
+                    "prefix":  "my_optional_data_docs_prefix",
+                },
+                "site_index_builder": {
+                    "class_name": "DefaultSiteIndexBuilder",
+                    "show_cta_footer": True,
+                },
+            }
+        },
+        validation_operators={
+            "action_list_operator": {
+                "class_name": "ActionListValidationOperator",
+                "action_list": [
+                    {
+                        "name": "store_validation_result",
+                        "action": {"class_name": "StoreValidationResultAction"},
+                    },
+                    {
+                        "name": "store_evaluation_params",
+                        "action": {"class_name": "StoreEvaluationParametersAction"},
+                    },
+                    {
+                        "name": "update_data_docs",
+                        "action": {"class_name": "UpdateDataDocsAction"},
+                    },
+                ],
+            }
+        },
+        anonymous_usage_statistics={
+          "enabled": True
+        }
+    )
+    ```
+
+
+2. **Pass this DataContextConfig as a project_config to BaseDataContext**
+
+    ```python
+    context = BaseDataContext(project_config=data_context_config)
+    ```
+
+3. **Use this BaseDataContext instance as your DataContext**
+
+    If you are using Airflow, you may wish to pass this Data Context to your GreatExpectationsOperator as a parameter. See the following guide for more details:
+
+    - [Deploying Great Expectations with Airflow](../../../guides/w) - **TODO(cdkini): Where do we link this?**
```

Comment:
Please note this TODO here; we do not have this file in our new docs.

https://docs.greatexpectations.io/en/latest/guides/workflows_patterns/deployment_airflow.html#deployment-airflow